### DISCUSSION TITLE
"builtin-baseline" code

### DISCUSSION THREAD

**Discussion by ache1077** [microsoft/vcpkg#26871] [General]
Hi
I have modified the vcpkg.json file and added 
`"builtin-baseline": "3426db05b996481ca31e95fff3734cf23e0f51bc"`

I would like to know where I can check the value of "builtin-baseline" for other versions

**Comment by Osyotr** [comment]
Baseline is simply a git commit hash. You can pick any valid commit in vcpkg repo for builtin baseline.
The value you mentioned points to commit https://github.com/microsoft/vcpkg/commit/3426db05b996481ca31e95fff3734cf23e0f51bc which is very-very old.

**Comment by e183228** [comment]
How does one get a more recent builtin-baseline?

**Reply by autoantwort** [reply]
`vcpkg x-update-baseline`

**Comment by ache1077** [comment]
You can find the most updated version at this address:

https://github.com/microsoft/vcpkg/commits/master
